### PR TITLE
Planwatch (fixes #67)

### LINKED ISSUE #67: "Planwatch" aka recently updated plans
This page needs to be created. You'll want to remove the placeholder for `recently_updated_plans` that I put in `routes.rb`.

### PR DESCRIPTION
**Pull request by PsychicNoodles**
Added the Planwatch page, which lists the most recently updated plans in a given past number of hours (default 12). This lives as a collection route under `plans`. Also changed the `rails_helper` for Planwatch from `recently_updated_plans` to `planwatch_plans`.

### CODE REVIEW

File: app/controllers/plans_controller.rb [hunk 1]
```diff
@@ -43,6 +43,11 @@ def search
     end
   end
 
+  def planwatch
+    @hours = params[:hours].to_i rescue 12
+    @plans = Account.where(changed: (Time.now - @hours.hours)..Time.now)
```
Comment (iangreenleaf) [inline app/controllers/plans_controller.rb]: Should these be ordered by `changed` as well?

**Comment by PsychicNoodles** [comment]
I added ordering by `changed` to the Account query.

**Comment by iangreenleaf** [comment]
Awesome, thanks. I'm gonna see if I can figure out what the deal is with the routes, but other than that this looks ready to go. Thanks!

**Comment by iangreenleaf** [comment]
One more request: could we set a default hours value so Plans show up when first loading the page?

**Comment by PsychicNoodles** [comment]
My default hour code was bugged -- didn't realize that Ruby/Rails converts a non-numeric string with `to_i` to 0, rather than throwing an error.

**Comment by iangreenleaf** [comment]
Fantastic, thanks!

**Pull request merged**